keyboards/anavi/macropad10/rules.mk

```diff
@@ -0,0 +1,9 @@
+# MCU name
+MCU = RP2040
+# Bootloader selection
+BOOTLOADER = rp2040
+
```

Comment:
```suggestion
```
These are already in info.json.

Reply:
Thank you for the code review. I have modified `rules.mk` and forced pushed the updated commit. Thanks, Leon